### PR TITLE
Setting up prometheus module to collect metrics

### PR DESCRIPTION
**Pull request by guilherme-carneiro-ca**
## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
We have only a initial structure from this project.

Issue Number: N/A

## What is the new behavior?
 - Created metrics route to expose all metrics.
 - Created config file to get API_KEY and APP_ID from command line.
 - Created scheduler file to schedule tasks via ['node-cron'](https://github.com/node-cron/node-cron).
 - Created index in prometheus folder to collect default metrics.
 - Added task to run test with [jest](https://jestjs.io/). 

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by merencia — APPROVED** [review]

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const server = require('./app/server');
+const router = require('./app/router');
+const prometheusPlugin = require('./plugins').prometheusPlugin;
+
```
Comment (cirocfc) [inline src/index.js:4]: blank line

File: src/plugins/prometheus/index.test.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+describe('prometheus index', () => {
+  const prometheusPlugin = require('./index');
+  const client = require('prom-client');
+  let scheduler = require('../../utils/scheduler');
+
+  beforeEach(() =>{
```
Comment (cirocfc) [inline src/plugins/prometheus/index.test.js]: missing space

File: src/utils/scheduler.test.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+describe('default chart', () => {
+  const scheduler = require('./scheduler');
+  const cron = require('node-cron');
+  const DEFAULT_INTERVAL = '* * * * *';
+
+  beforeEach(()=>{
```
Comment (cirocfc) [inline src/utils/scheduler.test.js]: missing spaces

File: src/utils/scheduler.test.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    cron.schedule = jest.fn();
+  });
+
+  test('should call cron.schedule', () => {
+    scheduler.start(()=>{});
```
Comment (cirocfc) [inline src/utils/scheduler.test.js]: missing spaces

**Review by cirocfc — APPROVED** [review]

**Pull request merged**